examples/README.md

```diff
@@ -0,0 +1,204 @@
+# Examples
+
+These gifs were created using [Charm VHS](https://github.com/charmbracelet/vhs).
+
+NOTE: there is some unresolved weirdness about how these examples render using VHS that are not quite
+the same as how TUI renders in most terminals.
+
+## Barchart ([barchart.rs](./barchart.rs)
+
+```shell
+cargo run --example=barchart --features=crossterm
+```
+
+![Barchart][barchart.gif]
+
+## Block ([block.rs](./block.rs))
+
+```shell
+cargo run --example=block --features=crossterm
+```
+
+![Block][block.gif]
+
+## Calendar ([calendar.rs](./calendar.rs))
+
+```shell
+cargo run --example=calendar --features=crossterm widget-calendar
+```
+
+![Calendar][calendar.gif]
+
+## Canvas ([canvas.rs](./canvas.rs))
+
+```shell
+cargo run --example=canvas --features=crossterm
+```
+
+![Canvas][canvas.gif]
+
+## Chart ([chart.rs](./chart.rs))
+
+```shell
+cargo run --example=chart --features=crossterm
+```
+
+![Chart][chart.gif]
+
+## Custom Widget ([custom_widget.rs](./custom_widget.rs))
+
+```shell
+cargo run --example=custom_widget --features=crossterm
+```
+
+This is not a particularly exciting example visually, but it demonstrates how to implement your own widget.
+
+![Custom Widget][custom_widget.gif]
+
+## Gauge ([gauge.rs](./gauge.rs))
+
+TODO: work out why the gauge background is rendering poorly via VHS.
```

Comment:
What do you think about making these `TODO`s comments?

```suggestion
<!-- TODO: work out why the gauge background is rendering poorly via VHS.-->
```

Reply:
I wanted to make this visible so that users viewing the README would understand that this is not the how this should render and it's a problem with VHS not Ratatui. I've raised a bug with VHS at https://github.com/charmbracelet/vhs/issues/344 for this

Reply:
I see. Maybe we should change it to "the background is rendering poorly when we automate generating these examples, it will render correctly when you run this example." or something like that?

Reply:
Updated the messages